packages/sns/src/governance.canister.ts

```diff
@@ -68,4 +71,32 @@ export class SnsGovernanceCanister extends Canister<SnsGovernanceService> {
     }
     return data.Neuron;
   };
+
+  /**
+   * Manage neuron. For advanced users.
+   */
+  manageNeuron = async (request: ManageNeuron): Promise<ManageNeuronResponse> =>
+    this.caller({ certified: true }).manage_neuron(request);
+
+  /**
+   * Set permissions of a neuron for a specific principal
+   */
+  addNeuronPermissions = async ({
+    neuronId,
+    permissions,
+    principal,
+  }: SnsAddNeuronPermissions): Promise<void> => {
```

Comment:
maybe it's interesting to return the `ManageNeuronResponse` here as well?

Particularly because it can contains the `{ Error: GovernanceError }`

Reply:
Good point! I forgot that it can return this error. Let me add it.

Reply:
I will raise the error if it's there

Reply:
Ah yeah, cool